src/inference/src/system_conf.cpp

```diff
@@ -100,7 +110,8 @@ bool with_cpu_x86_avx512_core_amx() {
     return with_cpu_x86_avx512_core_amx_int8() || with_cpu_x86_avx512_core_amx_bf16();
 }
 
-#else  // OPENVINO_ARCH_X86 || OPENVINO_ARCH_X86_64
+
+#else
```

Comment:
```suggestion
#else // OPENVINO_ARCH_X86 || OPENVINO_ARCH_X86_64
```
Otherwise clang-format job fails

Reply:
clang-format job is still failing - I believe because of extra space before `//`.
It's better to completely rollback your changes on these 2 lines.